docs/installation.md

```diff
@@ -2,38 +2,50 @@
- * [pytorch-geometric](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html): `conda install pyg -c pyg`
- * [Dependencies for pytorch geometric from wheels](https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html#installation-from-wheels): `pip install pyg_lib torch_scatter torch_sparse torch_cluster torch_spline_conv -f https://data.pyg.org/whl/torch-${TORCH}+${CUDA}.html`. 
-    - Here, `${TORCH}` and `${CUDA}` should be replaced by the pytorch and CUDA versions installed. You can find these using:
-      - `python -c "import torch; print(torch.__version__)"` and
-      - `python -c "import torch; print(torch.version.cuda)"`
-    - For example: `https://data.pyg.org/whl/torch-2.0.0+cpu.html`
- * Only if you have a MacOS with M1 chip, additional steps are needed:
-    * `conda install pytables`
-    * See [this](https://stackoverflow.com/questions/30145751/python3-cant-find-and-import-pyqt5) solution to install PyQt5 or run `conda install pyqt`
+Before installing deeprankcore you need to install some dependencies. We advise to use a [conda environment](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) with Python >= 3.9 installed. 
+
+* [msms](https://ssbio.readthedocs.io/en/latest/instructions/msms.html): `conda install -c bioconda msms`. *For MacOS with M1 chip users*: you can follow [these instructions](https://ssbio.readthedocs.io/en/latest/instructions/msms.html).
+* [DSSP 4](https://swift.cmbi.umcn.nl/gv/dssp/): 
+  * on ubuntu 22.04 or newer: `sudo apt-get install dssp`.
+    * If the package cannot be located, first run `sudo apt-get update`.
+  * on older versions of ubuntu or on mac, or lacking sudo priviliges: install from [here](https://github.com/pdb-redo/dssp), following the instructions listed.
+* [PyTorch](https://pytorch.org/):
+  * CPU only: `conda install pytorch==2.0.0 torchvision==0.15.0 torchaudio==2.0.0 cpuonly -c pytorch`
+  * if using GPU: `conda install pytorch==2.0.0 torchvision==0.15.0 torchaudio==2.0.0 pytorch-cuda=11.7 -c pytorch -c nvidia`
+* [pytorch-geometric](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html): `conda install pyg -c pyg`
+* [Dependencies for pytorch geometric from wheels](https://pytorch-geometric.readthedocs.io/en/latest/install/installation.html#installation-from-wheels): `pip install pyg_lib torch_scatter torch_sparse torch_cluster torch_spline_conv -f https://data.pyg.org/whl/torch-${TORCH}+${CUDA}.html`. 
+  - Here, `${TORCH}` and `${CUDA}` should be replaced by the pytorch and CUDA versions installed. You can find these using:
+    - `python -c "import torch; print(torch.__version__)"` and
+    - `python -c "import torch; print(torch.version.cuda)"`
+      - if this returns `None`, use `cpu` instead
+  - For example: `https://data.pyg.org/whl/torch-2.0.0+cpu.html`
+* Check if gcc is installed: `gcc --version`.
```

Comment:
Same comment as for the readme, in case you need to add info just to the same here